core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -316,6 +332,25 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Adds an {@link ChannelOption} to use in netty {@link ServerBootstrap} to create new {@link Server}.
```

Comment:
an {@link ChannelOption} -> a {@link ChannelOption}

Reply:
How about: `Adds a {@link ChannelOption} of {@link Server}'s server sockets.`